src/main/java/graphql/nadel/Nadel.java

```diff
@@ -45,11 +45,11 @@ private void init() {
 
         for (ServiceDefinition serviceDefinition : serviceDefinitions) {
             String serviceName = serviceDefinition.getName();
-            DelegatedExecution delegatedExecution = serviceDataFactory.getDelegatedExecution(serviceName);
+            ServiceExecution serviceExecution = serviceDataFactory.getDelegatedExecution(serviceName);
```

Comment:
Overall is ok. I tried to change some test cases locally to see that hydration works with field rename as well. I agree with Brad , need more testing here. Also I am not sure of that all test cases from Master are still here or we just remove them all. 
Another minor issue is that you change deletegatedExection to ServiceExection. but still have variables or methods with DelegatedExecution there.

Reply:
I added tests.
I agree the naming is not really there, but I will keep it for now, because I think ServiceExecution might not be the final name